### PR TITLE
https://jb-pythonic-py.github.io/ptCaseStudy/

### PR DESCRIPTION
**Pull request by JB-pythonic-py**

### CODE REVIEW

**Comment by JB-pythonic-py** [comment]
https://jb-pythonic-py.github.io/ptCaseStudy/

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <h3>Hiking</h3>
+                <p>Pacific Trails Resort has 5 miles of hiking trails and is adjacent to a state park.
+                    Go it alone or join one of our guided hikes.</p>
+            <h3>Kayaking</h3>
+                <p>Ocean kayaks are available for guest use.</p>
+            <h3>Bird Watching</h3>
+                <p>While anytime is a good time for bird watching at Pacific Trails, we offer guided
+                    birdwatching trips at sunrise several times a week.</p>
+            <br>
+        </main>
```
Comment (mbMosman) [inline activities.html]: Nice work with the cleanup on the main/footer.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+                <li>Guided hiking tours of the redwoods</li>
+            </ul>
+
+            <div id="contact">
+                <span Class="resort">Pacific Trails Resort</span>
+                <br>12010 Pacific Trails Road
+                <br>Zephyr, CA 95555
+                <br>
+                <br>888-555-5555
+                <br>
```
Comment (mbMosman) [inline index.html]: You don't need the extra `<br>` at the end now that we have padding on the footer.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+                    Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+            <h3>How are the yurts furnished?</h3>
+                <p>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove.
+                    The luxury camping experience also includes electricity and a sink with hot and cold running water.
+                    Shower and restroom facilities are located in the lodge.</p>
+            <h3>What should I bring?</h3>
+                <p>Bring a sense of adventure and some time to relax!
+                    Most guests also pack comfortable walking shoes and plan to dress
+                    for changing weather with layers of clothing.</p>
+            <br>
```
Comment (mbMosman) [inline yurts.html]: You don't need this extra `<br>`

File: activities.html [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+            <h3>Hiking</h3>
+                <p>Pacific Trails Resort has 5 miles of hiking trails and is adjacent to a state park.
+                    Go it alone or join one of our guided hikes.</p>
+            <h3>Kayaking</h3>
+                <p>Ocean kayaks are available for guest use.</p>
+            <h3>Bird Watching</h3>
+                <p>While anytime is a good time for bird watching at Pacific Trails, we offer guided
+                    birdwatching trips at sunrise several times a week.</p>
+            <br>
```
Comment (mbMosman) [inline activities.html]: You don't need this extra `<br>`

**Comment by mbMosman** [comment]
Ditch those extra br tags and it'll be perfect.  Nice work

**Pull request closed without merging**